src/test/java/com/openshift/internal/restclient/model/v1/RouteTest.java

```diff
@@ -63,7 +64,42 @@ public void createTLSConfigWhenUndefined() {
 		assertEquals("", tls.getKey());
 	}
 	
-	@Test
+    @Test
+    public void getPortWhenUndefined() {
+        ModelNode node = ModelNode.fromJSONString(Samples.V1_ROUTE_WO_TLS.getContentAsString());
+        route = new Route(node, client, ResourcePropertiesRegistry.getInstance().get(VERSION, ResourceKind.ROUTE));
+        ITargetPort port = route.getPort();
+        assertNull(port);
```

Comment:
Shouldn't we, in an additional test, also assert that the port, that's returned if you route.getPort().getTargetPort() is -1?